output/act-axe-result.json

```diff
@@ -1,870 +1,8665 @@
 [
   {
-    "ruleId": "SC1-1-1-image-has-name",
-    "axeIds": [
-      "image-alt"
-    ],
-    "ruleStatus": "automated",
-    "testCaseResults": [
-      {
-        "ruleId": "SC1-1-1-image-has-name",
-        "testcaseUrl": "https://auto-wcag.github.io/auto-wcag/auto-wcag-testcases/assets/SC1-1-1-image-has-name_passed_example_1.html",
-        "testcaseStatus": {}
-      },
-      {
-        "ruleId": "SC1-1-1-image-has-name",
-        "testcaseUrl": "https://auto-wcag.github.io/auto-wcag/auto-wcag-testcases/assets/SC1-1-1-image-has-name_passed_example_2.html",
-        "testcaseStatus": {}
-      },
-      {
-        "ruleId": "SC1-1-1-image-has-name",
-        "testcaseUrl": "https://auto-wcag.github.io/auto-wcag/auto-wcag-testcases/assets/SC1-1-1-image-has-name_passed_example_3.html",
-        "testcaseStatus": {}
-      },
-      {
-        "ruleId": "SC1-1-1-image-has-name",
-        "testcaseUrl": "https://auto-wcag.github.io/auto-wcag/auto-wcag-testcases/assets/SC1-1-1-image-has-name_failed_example_1.html",
-        "testcaseStatus": {}
-      },
-      {
-        "ruleId": "SC1-1-1-image-has-name",
-        "testcaseUrl": "https://auto-wcag.github.io/auto-wcag/auto-wcag-testcases/assets/SC1-1-1-image-has-name_failed_example_2.html",
-        "testcaseStatus": {}
-      },
-      {
-        "ruleId": "SC1-1-1-image-has-name",
-        "testcaseUrl": "https://auto-wcag.github.io/auto-wcag/auto-wcag-testcases/assets/SC1-1-1-image-has-name_failed_example_3.html",
-        "testcaseStatus": {}
-      },
-      {
-        "ruleId": "SC1-1-1-image-has-name",
-        "testcaseUrl": "https://auto-wcag.github.io/auto-wcag/auto-wcag-testcases/assets/SC1-1-1-image-has-name_inapplicable_example_1.html",
-        "testcaseStatus": {}
-      },
-      {
-        "ruleId": "SC1-1-1-image-has-name",
-        "testcaseUrl": "https://auto-wcag.github.io/auto-wcag/auto-wcag-testcases/assets/SC1-1-1-image-has-name_inapplicable_example_2.html",
-        "testcaseStatus": {}
-      },
-      {
-        "ruleId": "SC1-1-1-image-has-name",
-        "testcaseUrl": "https://auto-wcag.github.io/auto-wcag/auto-wcag-testcases/assets/SC1-1-1-image-has-name_inapplicable_example_3.html",
-        "testcaseStatus": {}
-      },
-      {
-        "ruleId": "SC1-1-1-image-has-name",
-        "testcaseUrl": "https://auto-wcag.github.io/auto-wcag/auto-wcag-testcases/assets/SC1-1-1-image-has-name_inapplicable_example_4.html",
-        "testcaseStatus": {}
+    "@context": {
```

Comment:
Do we need this 9,400 LOC file checked into Git?

Reply:
Sadly yes. We need this, as it is referenced from other projects like in `auto-wcag`